packages/venia-concept/src/components/MiniCart/product.js

```diff
@@ -22,14 +26,26 @@ const Product = props => {
 
     const classes = mergeClasses(defaultClasses, props.classes);
     const mask = isLoading ? <div className={classes.mask} /> : null;
-    const imageSource = useMemo(
-        () =>
-            resourceUrl(image.file, {
-                type: 'image-product',
-                width: imageWidth
-            }),
-        [image]
-    );
+
+    const productImage = useMemo(() => {
+        const src =
+            image && image.file
+                ? resourceUrl(image.file, {
+                      type: 'image-product',
+                      width: imageWidth,
+                      height: imageHeight
+                  })
+                : transparentPlaceholder;
```

Comment:
Out of curiosity, would we ever need a transparent placeholder that isn't 4x5?

Reply:
Related, would an alternate fix be to change our transparent placeholder to match the sample data ratio?

Reply:
> Out of curiosity, would we ever need a transparent placeholder that isn't 4x5?

Sure, and we can create/update those when we get to that point. All of the images (except for the landing page category "circle" images) _should_ be 4x5.

> Related, would an alternate fix be to change our transparent placeholder to match the sample data ratio?

I don't think it's worth it right now. @revanth0212 is going to investigate better ways to move forward with images/lazy-loading/etc and I think this may all change anyways.